packages/blade-coverage-plugin/background_script.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-undef */
+const calculateBladeCoverage = () => {
```

Comment:
Did not get this can you elaborate here ? Are you talking about script Anurag has written. If yes then this is duplicate. Usually this script will be modified once in 3 months may be and hence I am okay with duplication.